code/API_definitions/qod-api.yaml

```diff
@@ -1034,8 +1060,8 @@ components:
     EventQosStatus:
       description: |
         The current status of a requested or previously available session. Applicable values in the event are:
-        *  `AVAILABLE` - The requested QoS has been provided by the network
-        *  `UNAVAILABLE` - A requested or previously available QoS session is currently unavailable. `statusInfo` may provide additional information about the reason for the unavailability.
+        *  `AVAILABLE` - The requested QoS has been provided by the network. NOTE: this event will be sent always, also if status within the response to `createSession` is already `AVAILABLE`
```

Comment:
I don't understand this note.

Does this mean that "AVAILABLE will always be sent if the status of `createSession` is also `AVAILABLE`"

Reply:
I agree that this note is not understandable. I've deleted it as the (to be) expected behaviour is already described within the documentation of 'createSession'.

Reply:
Done